pysat/_instrument.py

```diff
@@ -2004,14 +2000,8 @@ def custom_apply_all(self):
                         recast_as_dict = False
                         if not isinstance(new_data, dict):
                             new_data = {'data': new_data}
-                            # dstr = ''.join(('Data returned should be a dict ',
```

Comment:
This didn't last long. Technically you just approved it the other day :P

After our discussion on Tue. this deprecation warning will be effectively moving to the custom_attach for anything other than 'modify'.

Reply:
Yeah, but then we had our discussion!  We've gotta stay on our toes :)